doc/applications_and_porting.rst

```diff
@@ -27,3 +27,75 @@ Required features of the hardware
 
 * It should be possible to send and receive raw (layer 2) Ethernet frames
 * It should be possible to store data between runs, in a file system or some nonvolatile memory
+
+
+Minimum cycle time for your application and hardware
+----------------------------------------------------
+At PLC configuration, there are two adjustable parameters regarding the
+cyclic data timing for an IO-device. One basically controls the
+frame-to-frame transmission interval (given as a mulitple of the cycle time),
+and the other one controls the PLC watchdog timeout setting.
+The watchdog triggers an alarm when a number of consecutive cyclic data
+frames from the IO-device are missing.
+
+During conformance testing, the PLC watchdog timeout setting is 3 x
+frame-to-frame interval times, ie the PLC will trigger an alarm if three
+consecutive cyclic data frames are missing. Thus the smallest allowed
+frame-to-frame interval given in the GSDML file must take the smallest
+useful PLC watchdog timeout setting into account.
+
+There can be different factors limiting the ability to send Profinet frames at
+a regular pace:
+
+* The initial work at Profinet communication startup should not cause
+  delays in the cyclic data transmission once started.
+* The hardware needs to keep up with the steady-state sending and receiving
+  of Profinet cyclic data.
+* Other work done by the Profinet application, for example work done in
+  callbacks, might slow down the communication.
+* Operating system jitter, which basically is that the operating system is
+  busy doing other work, will cause frame delays.
+* Using a network interfaces connected via USB can cause additional frame
+  delays.
+
+For real-time operating systems (RTOS) for example rt-kernel, the
+frame-to-frame transmission will have little jitter. The amount of work
+needs to be done during Profinet communication setup or frame transmission is
```

Comment:
_that_ needs to be done?

What does this refer to? If there is a timing problem during startup then that may just be a problem with the current implementation.

Reply:
I removed "The amount of work ..." as it is not really adding any information.
Done.